BUILD.md

```diff
@@ -117,24 +117,32 @@ Create a .rpm:
 The following steps must be completed once. Make sure you run them in the
 specified qubes.
 
-#### In `dom0`
+Overview of the qubes you'll create:
 
-1. Create a new Fedora **template** (`fedora-37-dz`) for Dangerzone development:
+>    | qube         |   type   | purpose |
+>    |--------------|----------|---------|
+>    | dz           | app qube | Dangerzone development |
+>    | dz-dvm       | app qube | offline diposable template for performing conversions
+>    | fedora-38-dz | template | template for the other two qubes
```

Comment:
Two quick comments here:

1. I think we can present the table without the block quotes.
2. You've left some `|` markers out of the last two lines (see end of line). GitHub still renders the table, but we could be more strict.

Reply:
Done in [af1ffac](https://github.com/freedomofpress/dangerzone/pull/533/commits/af1ffacc5838fdc37986a6fb0ef2a561c8ed8017)